example/example_test.go

```diff
@@ -9,26 +9,23 @@ import (
 )
 
 func TestExample(t *testing.T) {
-
-	//in order to run a test that will reset to known state in setup_test use the `ot.Run(t,...)``method instead of `t.Run(...)`
+	// in order to run a test that will reset to known state in setup_test use the `ot.Run(t,...)``method instead of `t.Run(...)`
 	ot.Run(t, "Example test", func(t *testing.T) {
 		block, err := ot.O.GetLatestBlock(context.Background())
 		require.NoError(t, err)
-		assert.Equal(t, 6, int(block.Height))
+		assert.Equal(t, 4, int(block.Height))
```

Comment:
How come the reduction in block height? I didn't see anything relevant in the changes on this PR. Is it from the dependencies?

Reply:
I do not know, i do nothing before this in the test so there has to be less stuff happening in the emulator before i get it.

Reply:
Makes sense :+1: